update-action/footer.html

```diff
@@ -1,17 +1,17 @@
 <footer class="site-foot">
 	<p class="socials-wrapper-centered">
-		<a href="https://x.com/tbfighters" target="_blank">
-			<img src="{prefix}img/socials/twitter.svg" alt="Twitter" class="social-icon">
-			<img src="{prefix}img/socials/twitter-hover.svg" alt="Twitter" class="social-icon-hover"></a>
-		<a href="https://www.instagram.com/tbfighters" target="_blank">
-			<img src="{prefix}img/socials/insta.svg" alt="Instagram" class="social-icon">
-			<img src="{prefix}img/socials/insta-hover.svg" alt="Instagram" class="social-icon-hover"></a>
-		<a href="https://www.youtube.com/@TB_Fighters" target="_blank"><img
-				src="{prefix}img/socials/youtube.svg" alt="Youtube" class="social-icon"><img
-				src="{prefix}img/socials/youtube-hover.svg" alt="Youtube" class="social-icon-hover"></a>
-		<a href="https://www.facebook.com/profile.php?id=61555332203505" target="_blank"><img
-				src="{prefix}img/socials/facebook.svg" alt="Facebook" class="social-icon"><img
-				src="{prefix}img/socials/facebook-hover.svg" alt="Facebook" class="social-icon-hover"></a>

```

Comment:
The {prefix} part is needed for sub pages like the ones in the template directory